src/encoded/batch_download.py

```diff
@@ -140,8 +140,45 @@ def includeme(config):
     ('Restricted', ['files.restricted'])
 ])
 
+_tsv_mapping_publicationdata = OrderedDict([
+    ('File accession', ['files.title']),
+    ('File dataset', ['files.dataset.accession']),
+    ('File type', ['files.file_format']),
+    ('File format', ['files.file_type']),
+    ('File output type', ['files.output_type']),
+    ('Assay term name', ['files.assay_term_name']),
+    ('Dataset accession', ['accession']),
+    ('Dataset date released', ['date_released']),
+    ('Project', ['award.project']),
+    ('Lab', ['files.lab.title']),
+    ('md5sum', ['files.md5sum']),
+    ('dbxrefs', ['files.dbxrefs']),
+    ('File download URL', ['files.href']),
+    ('Assembly', ['files.assembly']),
+    ('Controlled by', ['files.controlled_by']),
+    ('File status', ['files.status']),
+    ('Derived from', ['files.derived_from']),
+    ('S3 URL', ['files.cloud_metadata.url']),
+    ('Size', ['files.file_size']),
+])
```

Comment:
can we add the following?
`  ('Biosample term id', ['biosample_ontology.term_id']),
    ('Biosample term name', ['biosample_ontology.term_name']),
    ('Biosample type', ['biosample_ontology.classification']),
    ('Target', ['target.label']),
    ('No File Available', ['file.no_file_available']),
    ('Restricted', ['files.restricted'])
`
We really need organism, but I guess it's not in File.

Reply:
All batch_download versions filter out No File Available=true files, so I left it out.

Reply:
puzzling why it's in the TSV specifier then

Reply:
I don‘t think any target is available — not in the file nor the PublicationData

Reply:
https://www.encodeproject.org/search/?type=File&target.label=*
https://www.encodeproject.org/search/?type=File&biosample_ontology.term_name=*

They are not always there, but they are there if they can be calculated.

Reply:
you are correct, publication_data doesn't have anything, which is fine.